### ISSUE TITLE
Freeze

### ISSUE THREAD

**Issue by zRiaz** [drtshock/Factions#164]
Pretty much all commands causes the server to freeze for about 2 seconds.
(/f who, show, leave create, setwarp)
Using latest version on Spigot

**Comment by drtshock** [comment]
How many players do you have on? And which latest version of spigot?

**Comment by zRiaz** [comment]
1.8 spigot only me

**Comment by zRiaz** [comment]
it was on a testing server

**Comment by drtshock** [comment]
Try it on 1.7.10

**Comment by zRiaz** [comment]
Will try tomorrow, if this turns out to be a 1.8 issue how long would you estimate it to take to fix it? Got a 1.8 map reset in 2 weeks so yeh..

**Comment by zRiaz** [comment]
Ok @drtshock I have figured out why this is happening. However I have not tested on 1.7.
It only happens when Economy is set to true in the conf.json and when its set to false everything works fine.

**Comment by Kakifrucht** [comment]
Now that you mention it, this happens to me, too. This has nothing to do with the economy though.

Basically the first time this appeared is when the chat hover stuff was added. A fix would be to do the fetching of all the information about enemies, allies and players of a faction async, as this makes the server very laggy when it comes to bigger factions (and I'm running on an i7 3770k). I'm still on spigot 1.7.10 (Factions build no. 167), using a command like /f who whateverunexistantfaction and the server will lagg  heavily until it prints an exception:

http://pastebin.com/UyLiixHa

But even if it doesn't print an exception, it laggs slightly whenever you use such commands. Leaving/disbanding makes it quite laggy imo, creating doesn't.

Here a short Timings file I took while executing /f who three times. http://timings.aikar.co/?url=9521073

I hope for a quick async fix!
